modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -73,7 +104,7 @@ apt install -y \
   php-smbclient coreutils php-ldap
 ----
 
-WARNING: Ubuntu 18.04 includes smbclient 4.7.6, which has a known limitation of only using version 1 of the SMB protocol.
+WARNING: Ubuntu {ubuntu-version} includes smbclient 4.7.6, which has a known limitation of only using version 1 of the SMB protocol.
 
```

Comment:
I disagree with replacing 18.04 with a variable here. ASFAIK this limitation is only in 18.04

Reply:
Fair enough.

Reply:
I a agree with voroyam to disagree. smbclient version number is specific to 18.04, document name is specific to 18.04

Reply:
In the most recent commit, https://github.com/owncloud/docs/pull/2480#discussion_r394988085 has been addressed.